package.json

```diff
@@ -36,7 +36,8 @@
     "react-helmet": "^5.2.1",
     "react-icons": "^3.7.0",
     "reactstrap": "^8.0.1",
-    "styled-components": "^4.4.0"
+    "styled-components": "^4.4.0",
+    "yarn": "^1.21.1"
```

Comment:
huh?

Reply:
Added accidentally.

Reply:
Alright - can we remove it? Then I'm happy to merge

Reply:
should be resolved now.